### DISCUSSION TITLE
Back up to multiple destinations in the same profile

### DISCUSSION THREAD

**Discussion by strmd** [borgbase/vorta#853] [Ideas]
It would be very nice to be able to be able to manage profiles more directly. Here are some ideas:
- Import/export - repo-specific, single file (obviously shouldn't contain passphrase or such).
- Ability to duplicate profiles - for applying the same profile to multiple targets, e.g. two offsite drives.
- Templates (possibly) - save profile as template, create profile from template. Repo-agnostic. Perhaps expand default settings to a default template to be used as starting point for novice users, i.e. 'back up home folder except [on macOS: .Trashes, **/.DS_Store] with these sensible prune options', or onboarding offer to install a few example templates.

**Comment by m3nu** [comment]
Import/export would be easy since every profile is just a row in sqlite.

Instead of duplication, I'd suggest to have backups to multiple repos in a single profile. Borgmatic does it like this. Of course it would need to fail gracefully if the destination isn't available.

Personally I have 2 HDs for onsite backups (using the same source folders and exclusions). I find myself changing the repo and then running the backup again. It sounds like your use-case is the same.

**Comment by strmd** [comment]
> Instead of duplication, I'd suggest to have backups to multiple repos in a single profile.

That might work just as well, yes. The profile as a starting point is common across other software and might be easier to grasp (I'm thinking of Vorta as a way to get people with no CLI experience to adopt Borg) but doing it in the way that makes most sense from a Borg perspective might be more flexible. Multiple destinations in the same profile would open up some neat possibilities, i.e. back up to every available destination, local or remote.

> Personally I have 2 HDs for onsite backups (using the same source folders and exclusions). I find myself changing the repo and then running the backup again. It sounds like your use-case is the same.
Indeed. I did make two profiles and named them accordingly, but all settings including the password are identical.

**Comment by m3nu** [comment]
Then let's start out with multiple repos in one profile. For implementation, the repo-dropdown could have [checkboxes](https://stackoverflow.com/questions/22775095/pyqt-how-to-set-combobox-items-be-checkable), like this:

![llcc0](https://user-images.githubusercontent.com/3916435/50468300-0739d400-09e2-11e9-81bd-4f4fbc0f6214.png)

**Comment by strmd** [comment]
👍

**Comment by mauromol** [comment]
Multiple destinations for the same profile would be very useful, indeed. I really hope you can add this feature to Vorta.

Just a little note: having the ability to "clone"/"duplicate" a profile would still be useful when you want to create a new profile similar to an existing one but with few changes, instead of recreating it from scratch.

**Comment by PaulWoitaschek** [comment]
I'd love to see multiple repos for one profile too. Currently I need to manually sync my local and remote profile.

**Reply by m3nu** [reply]
Thanks for weighing in. Agree that this will be handy. Added a new issue #942 to track progress.
